src/api/uploads/MultiputPart.js

```diff
@@ -265,7 +262,7 @@ class MultiputPart extends BaseMultiput {
                 throw new Error('logEvent endpoint not found');
             }
 
-            await retryNumOfTimes(
+            retryNumOfTimes(
```

Comment:
Does this still belong in the `try/catch`?

Reply:
Good point, I'll refactor this to attach a log on error instead